index.d.ts

```diff
@@ -78,29 +78,27 @@ declare module 'gdax' {
     }
 
     export type PageArgs = {
```

Comment:
One of these is required — shouldn't that be enforced?

Reply:
Currently, I am using `getOrders` and `getFills` without specifying any of those parameter, and the result I get back is reasonably correct. I am not sure those parameters are mandatory; am I wrong?

Reply:
Right, they are only required if you want to access another page :D